apps/web/models.py

```diff
@@ -33,11 +35,13 @@ class Team(models.Model):
 
     name = models.CharField(max_length=100)
     email = models.EmailField(max_length=70)
+    description = RichTextField(null=True)
```

Comment:
I guess @aka-jain mentioned explicitly for a text editor in admin, so, I used ckeditor. Well, will be reverting if not needed :)